src/Rule/Nested.php

```diff
@@ -238,8 +271,22 @@ private function normalizeRules(): void
 
     public function propagateOptions(): void
     {
+        if ($this->rules === null) {
+            return;
+        }
+
         $rules = [];
+        /** @var iterable<RuleInterface> $attributeRules */
```

Comment:
Seems. property `$rules` should be `RuleInterface[]|null`.

Reply:
Iterable is also possible.

- Moved PHPDoc closer to where it's used.
- Added `null` in PHPDoc.
- Added handling of `null` for a loop.